_templates/exchanges.html

```diff
@@ -417,11 +417,8 @@ <h3 id="south-africa" class="no_toc">South Africa</h3>
 
 
 <p class="introlink exchanges-introlink exchanges-buy-link">Visit
-  <a href="https://www.buybitcoinworldwide.com/">Buy Bitcoin Worldwide</a> for user reviews on some of the above exchanges.
+  <a href="https://www.buybitcoinworldwide.com/">Buy Bitcoin Worldwide</a> for user reviews on some of the above exchanges, or <a href="https://cryptoradar.co">Cryptoradar</a> for comparison based on prices, fees and features.
```

Comment:
@josiasmontag Could you please make the word "comparison" pluralized?

Reply:
Done